src/gui/log/logmodel.cpp

```diff
@@ -72,6 +73,7 @@ QVariant BaseLogModel::Message::type() const
 BaseLogModel::BaseLogModel(QObject *parent)
     : QAbstractListModel(parent)
     , m_messages(MAX_VISIBLE_MESSAGES)
+    , m_timeForeground(UIThemeManager::instance()->getColor(QLatin1String("Log.Time"), Qt::darkGray))
```

Comment:
I would use "Timestamp" instead of "Time".

Reply:
Can you take quick look at another PR as well, I got selected in GSoC😎 and has upcoming exams, so it would very nice if you and @Chocobo1 can review these today.